pages/common/tldr.md

```diff
@@ -7,14 +7,14 @@
 
 `tldr {{command}}`
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Show the tldr page for `cd`, overriding the default platform(if the client supports -p):
 
 `tldr -p {{windows}} {{cd}}`
 
-- Show the tldr page for a subcommand:
+- Show the tldr page for a sub-command:
```

Comment:
small error, feel free to change this back

Reply:
I'm not sure about this — if it's really more correct, then we should change it across all of the pages.